pkg/bindings/test/common_test.go

```diff
@@ -20,9 +20,18 @@ type testImage struct {
 }
 
 const (
+	devPodmanBinaryLocation     string = "../../../bin/podman"
 	defaultPodmanBinaryLocation string = "/usr/bin/podman"
 )
 
+func getPodmanBinary() string {
+	_, err := os.Stat(devPodmanBinaryLocation)
+	if os.IsNotExist(err) {
+		return defaultPodmanBinaryLocation
+	}
+	return devPodmanBinaryLocation
```

Comment:
On permission denied errors, you are going to return devPodmanBinaryLocation

Reply:
i did that on purpose ... thats ok to me